spring-cloud-aws-autoconfigure/src/main/java/org/springframework/cloud/aws/autoconfigure/micrometer/AwsEc2MicrometerMetricsProperties.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2013-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.aws.autoconfigure.micrometer;
+
+import java.util.Collection;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import static java.util.Collections.EMPTY_LIST;
+
+/**
+ * @author Renan Reis Martins de Paula
+ */
+@ConfigurationProperties(prefix = "cloud.aws.ec2.micrometer.metrics")
```

Comment:
Are these metrics specific to EC2? Would they will be also sent when running on ECS or EKS? I think the configuration properties prefix should be `cloud.aws.cloud-watch`.

Reply:
Since the properties are injected in the Environment by AmazonEc2InstanceDataPropertySource I assumed it is specifically for EC2.